src/wymeditor/parser/xhtml-sax-listener.js

```diff
@@ -172,9 +172,22 @@ WYMeditor.XhtmlSaxListener = function() {
 };
 
 WYMeditor.XhtmlSaxListener.prototype.shouldCloseTagAutomatically = function(tag, now_on_tag, closing) {
-    closing = closing || false;
+    if (tag != 'td' && tag != 'option') {
```

Comment:
We're only handling autoclosing for these two things because these are two especially problematic tags to leave open. A full solution would work with all of the tags that the HTML5 spec says should autoclose, but we're trying to avoid being a (bad) re-implementation of [parse5](https://github.com/inikulin/parse5). We're doing this because we found a bug in production with nested tables caused by this attempt at a fix.